### ISSUE TITLE
[Designer] Bug 31615058: [Usable - Adaptive Cards - WeatherCompact.Json]: Narrator focus jumps degree F separately in scan mode by using up and down arrow key.

### ISSUE THREAD

**Issue by aaagrawal** [microsoft/AdaptiveCards#7629]
### Problem Description

[Bug 31615058](https://microsoft.visualstudio.com/OS/_workitems/edit/31615058): [Usable - Adaptive Cards - WeatherCompact.Json]: Narrator focus jumps degree F separately in scan mode by using up and down arrow key.

User Experience:

Screen reader users will be impacted if Narrator focus jumps degree F separately in scan mode by using up and down arrow key.

Test Environment:

Application: Adaptive Cards
OS version: 2004 (OS Build 21301.1000)

Application Version: 2101.28.1.0

Pre-Requisite: Launch the Adaptive Card application.

Repro steps:

1. Launch the Adaptive Card application and Enable Narrator.

2. Adaptive Card UWP Visualizer screen starts appearing. 

3. Navigate to WeatherCompact.Json button and activate it.

4. Adaptive card will be appeared on right sight of the pane.

5. Navigate the adaptive card by using down arrow key in scan mode and verify the issue.

Actual Result:

Narrator focus jumps 46 and degree F separately in scan mode by using up and down arrow key and narrator announces degree F.

Expected result:

Narrator focus should jump together on 46 and degree F and Narrator should announce 46 degree Fahrenheit.

____Same bug found on Android FC-Jan22 Test Pass____

Adding <HCL-AdaptiveCards-Android> Tag to this bug.

Test Environment:
OS: Android 12
Device: S21+
Screen reader: Talkback
AdaptiveCard Android App version: 2021.10.15.1 (40331737)

Issue faced: Talkback focus moves to 46 and degree F separately navigating via swipe getsures.

### Card JSON

```JSON
Also, repro on UWP latest release
```
